content/pages/processing/conversion.rst

```diff
@@ -0,0 +1,3 @@
+DICOM Conversion
```

Comment:
maybe the file name "dicom" would be a better fit if this page is only about dicom conversions.

Reply:
Generally, I agree. But again depends on what is to come in this section. If we are going to have "dicom conversion" and some "dicom metadata extraction", then just "dicom.rst" wouldn't be better. ;-)

Reply:
My only point was that "conversion" is very broad (not dicom specific). Whereas if all the info will be about dicoms (like conversion, metadata extraction, etc), then perhaps "dicom" would be a better name.

In all, it's a nitpick. Not something that I'm invested in.

Reply:
Sure. What I meant by "in this section" would be a page for "dicom conversion" and another one for "dicom metadata extraction", both under "data processing". In that case we would need to change the name again. So I'd simply change it, whenever we know what else is to be build in here.

Reply:
ACK